.github/workflows/python-plugins.yml

```diff
@@ -0,0 +1,70 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test plugins
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "*" ]
+    types:
+    - labeled
+    - opened
+    - synchronize
+    - reopened
+
+
+jobs:
+  build:
+    if: contains( github.event.pull_request.labels.*.name, 'test-plugins')
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.9]
+        plugin-name:
+        - integral
+        - integral-all-sky
+        - antares
+        # - polar
+
+    services:
+      antares-backend:
+        image: densavchenko/antares:5429d35
+        ports: 
+        - 5002:8000
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+
+    - name: Clone ${{ matrix.plugin-name }} plugin 
+      uses: actions/checkout@v2
+      with:
+        repository: 'oda-hub/dispatcher-plugin-${{matrix.plugin-name}}'
+        path: plugin   
+
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip flake8 pytest mypy pylint pytest-cov    
+        
+        if [ -f plugin/requirements.txt ]; then 
+          < plugin/requirements.txt grep -v oda-hub/dispatcher-app | grep -v cdci-data-analysis | grep -v oda_api > plugin-req.txt
+          pip install -r plugin-req.txt ./plugin[test] .[test]; 
+        fi        
```

Comment:
does this mean that the `plugin/requirements.txt` file will have to be included? Or is it created from one of the previous step? I am not familiar yet with github workflows

Reply:
yes, it is created in the other step. Plugin is cloned into `plugin` directory. This works.

Some tests fail for `integral` plugin. Strange. I look.

Reply:
That's unrelated. But strange.